src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -110,16 +113,17 @@ public function initialize(array $config)
      * @param string $model the model field value
      * @return void
      */
-    public function setupFieldAssociations($fields, $table, $model)
+    public function setupFieldAssociations($fields, $table, $model, $strategy)
     {
+        $targetAlias = Inflector::slug($table);
```

Comment:
Couldn't you use `underscore()` instead of slug? Or do you expect non-ascii characters in the table names?